src/util/Constants.js

```diff
@@ -73,6 +73,8 @@ exports.Endpoints = {
         makeImageUrl(`${root}/team-icons/${teamId}/${hash}`, { size, format }),
       Sticker: (stickerId, stickerFormat) =>
         `${root}/stickers/${stickerId}.${stickerFormat === 'LOTTIE' ? 'json' : 'png'}`,
+      // TODO: Need more info
+      RoleIcon: (roleId, hash, format) => `${root}/role-icons/${roleId}/${hash}.${format === 'png' ? 'png' : 'jpeg'}`,
```

Comment:
The upstream PR mentioned png/jpeg only but it seems like you can set other formats too so this should be updated to use makeImageUrl instead
https://cdn.discordapp.com/role-icons/860078619648851978/cf3ced8600b777c9486c6d8d84fb4327.webp?size=64
Also did you miss the Role#iconURL method or is this coming later?

Reply:
yup, I wasn't sure about the info that was available at the moment so I just kept it like that. I did forget about the method tho, so thanks for that.

Reply:
This can be resolved then I guess